RPGManager_development/src/rpg_database/character_sheet/LanguagesSet.java

```diff
@@ -0,0 +1,110 @@
+package rpg_database.character_sheet;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Set;
+
+import rpg_database.character_sheet.interfaces.CustomSetter;
+
+public class LanguagesSet implements Set<Languages>, CustomSetter<LanguagesSet>{
+	
+	private final HashSet<Languages> languagesSet;
+	
+	public LanguagesSet(Languages...languages){
+		languagesSet = new HashSet<Languages>();
+		languagesSet.addAll(Arrays.asList(languages));
+	}
+	@Override
+	public Class<LanguagesSet> getImplementingClass() {
+		return LanguagesSet.class;
+	}
+	
+	@Override
+	public void setSelfInSheet(CharacterSheet characterSheet) {
+		LanguagesSet temp = (LanguagesSet) characterSheet.characterData.get(Fields.LANGUAGES);
+		temp.addAll(this);

```

Comment:
A setData-nak mindig felül kell írnia az előző objektumot az átadott új objektummal. Ne a setData-val adjunk hozzá az előzőhöz....
Cserébe látom hogy ezt a tesztekben is így használod. Ha hozzá szeretnél adni a nyelvekhez akkor ahhoz kérd el a LanguagesSet objektumot és ahhoz adj hozzá a tesztben majd légyszi.